### ISSUE TITLE
Godot doesn't display all files included in a build in the FileSystem

### ISSUE THREAD

**Issue by blockimperiumdao** [godotengine/godot#84868]
### Godot version

v4.1.2.stable.official [399c9dc39]

### System information

Godot v4.1.2.stable - macOS 14.0.0 - Vulkan (Forward+) - integrated Apple M1 Max - Apple M1 Max (10 Threads)

### Issue description

When building a project, there will be times when certain resource files (like HTML files) need to be included in a build. While one can go into an export and add *.html files to that export, the files themselves will not show up in the editor - so it is not clear that they are effectively in a project and it is cumbersome to create paths to them since we can't see them in the FileSystem.

TLDR

Please add all files that would be included in a build into the FileSystem so that we can see them easily, open them with external editors, etc.

### Steps to reproduce

Create a new project. Add an HTML file to the file system, the file will not show up in the FileSystem. Go into the export and add the *.html to the exports (and it would be nice to have a Global setting across projects - but that will be a feature request) and the files still don't show up in the FileSystem.

### Minimal reproduction project

N/A

**Comment by Calinou** [comment]
Thanks for the report! Consolidating in https://github.com/godotengine/godot-proposals/issues/677.
